.github/actions/go-setup-cache/action.yml

```
@@ -37,8 +37,7 @@ runs:
     - name: Determine Golang cache paths
       id: golang-path
       run: |
-        echo "::set-output name=build::$(go env GOCACHE)"
-        echo "::set-output name=module::$(go env GOMODCACHE)"
+        echo "build=$(go env GOCACHE)" | tee -a $GITHUB_OUTPUT
```

I think this is missing the `module` var?
```suggestion
        echo "build=$(go env GOCACHE)" | tee -a $GITHUB_OUTPUT
        echo "module=$(go env GOMODCACHE)" | tee -a $GITHUB_OUTPUT
```

---

Woops, good catch.